internal/app/connectconformance/testsuites/grpc-web-client.yaml

```diff
@@ -0,0 +1,576 @@
+name: gRPC-web
+relevantHttpVersions:
+  - HTTP_VERSION_1
+relevantCodecs:
+  - CODEC_PROTO
+relevantProtocols:
+  - PROTOCOL_GRPC_WEB
+relevantCompressions:
+  - COMPRESSION_IDENTITY
```

Comment:
Should we only run with identity?

Reply:
Seems like you don't really need to be so specific with these relevant aspects of these cases since they are redundant with what's in the supported features in `testdata/grpc-web-client-impl-config.yaml`.

Reply:
It seems that it does support compression by setting the `Content-Encoding` header and letting the browser do the compression. I updated the tests and the client implementation to correctly set this header under the various compression scenarios.

In addition, though, I made a change to the referenceserver checks for the `Content-Encoding` header. According to [this doc](https://github.com/grpc/grpc-web/blob/master/doc/browser-features.md#compression), gRPC-web seems to use `Content-Encoding` and ignores `Grpc-Encoding`.

Reply:
> It does support compression, but I think we have an inconsistency with Connect via gRPC-web vs. the gRPC-web client.

Yeah, just skip the compression. Connect is doing things the right way. I guess the grpc-web Envoy plugin supports full-body compression, but that is not actually part of any of the specs, so fine to just skip compression entirely for this client.